doc/DeveloperGuide.md

```diff
@@ -87,6 +87,28 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Renaming of an existing tag
+
+**MSS**
+
+1. User asks to list all tags
```

Comment:
Great but you do need to think about the minimal scenario such that this use case would still be comprehensible i.e. which steps can be omitted yet the sequences still make sense